src/command/build.rs

```diff
@@ -255,6 +255,12 @@ impl Build {
         Ok(())
     }
 
+    /// Returns local and latest wasm-pack versions.
+    pub fn return_wasm_pack_versions() -> Result<(String, String), Error> {
```

Comment:
I realize I am late to the party but personally I'd like too see this function return a struct that contains `local` and `latest`. Because I think it's not clear for the caller which `String` is which. So maybe something like:

```rust
struct WasmPackVersion {
  local: String,
  latest: String,
}
```

Reply:
That seems like a good improvement, but also somethign that could potentially happen in a follow up PR (if that is easier for @daubaris)

Reply:
Yeah, I agree.